packages/CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+(modification: no type change headlines) and this project adheres to
+[Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [2.0.0] - 2020-08-25
+
+This project got a major overhaul, with several updates to the tooling, including packages renaming and retiring some scripts. Below is a breakdown by package.
+
+**@ethereumjs/config-nyc => [@ethereumjs/coverage](https://github.com/ethereumjs/ethereumjs-config/tree/master/packages/coverage)**
```

Comment:
Can we keep the `config` here in the package name (so `@ethereumjs/config-coverage`)? The naming is already somewhat inconsistent by this `eslint-config-defaults`  naming (this was due to some ESLint limitation if I recall correctly?).

Reply:
Have nevertheless approved, maybe we can update on the release PR itself.

Reply:
@holgerd77 in fact, that was a typo on the Changelog. The package name is correctly set to `@ethereumjs/config-coverage` 👍